app/ami.py

```diff
@@ -209,10 +209,11 @@ def estimate_monthly_ami_price(ami_type: str, block_device_mappings: list, ami_n
 
 
 def is_ami_registered(ami_id: str, ) -> bool:
```

Comment:
Seems to be an extra `,`
```suggestion
def is_ami_registered(ami_id: str) -> bool:
```